tenders/hvt/hvt_main.c

```diff
@@ -196,5 +196,11 @@ int main(int argc, char **argv)
 
     setup_modules(hvt);
 
+#if(HVT_DROP_PRIVILEGES == 1)
+    hvt_drop_privileges();
+#else
+    warnx("Stern Warning, not recommended for production");
```

Comment:
`WARNING: Tender is configured with HVT_DROP_PRIVILEGES=0. Not dropping any privileges.`
`WARNING: This is not recommended for production use.`